pkg/repo/mysql/mysql.go

```diff
@@ -0,0 +1,66 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/XSAM/otelsql"
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/spf13/viper"
+	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
```

Comment:
v1.4.0 isn't there are newer version?

Reply:
vscode automatic import, is silly sometimes, but you are right